projects/ion/src/lib/icon/svgs/icons.ts

```diff
@@ -0,0 +1,7 @@
+export const iconsPaths = {
```

Comment:
Do you think that is necessary to add a comment here to describe how to add an icon? Something like:
```suggestion
// To add an icon, use kebab-case on key name and insert only SVG paths on value
export const iconsPaths = {
```